guides/common/modules/proc_uploading-additional-scap-content-using-cli.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="uploading-additional-scap-content-using-cli"]
+= Uploading additional SCAP content using CLI
+
+You can upload additional SCAP content into {ProjectServer}, either content created by yourself or obtained elsewhere.
+ifdef::satellite[]
+Note that {Team} only provides support for SCAP content obtained from {Team}.
```

Comment:
```suggestion
{Team} only provides support for SCAP content obtained from {Team}.
```

AFAIK it's either an admonition or we drop "note that".

Reply:
https://github.com/theforeman/foreman-documentation/issues/4214